src/openrct2/peep/Guest.cpp

```diff
@@ -3596,10 +3596,30 @@ void rct_peep::UpdateRideAdvanceThroughEntrance()
             peep_update_ride_leave_entrance_maze(this, ride, entranceLocation);
             return;
         }
-        Guard::Assert(ride->type == RIDE_TYPE_SPIRAL_SLIDE);
```

Comment:
Do you have a save this hits this assert in develop and is fixed by this PR?

Reply:
I've attached the park to the first post.